### PR TITLE
feat: add taskdetails component

### PR DESCRIPTION
**Pull request by NeilChavez**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "@emotion/styled": "^11.11.0",
     "@mui/material": "^5.13.2",
     "@reduxjs/toolkit": "^1.9.5",
+    "pnpm": "^8.5.1",
```
Comment (Alan-TheGentleman) [inline package.json]: pnpm shouldn't be a dependency

File: package-lock.json
Comment (Alan-TheGentleman) [inline package-lock.json:1]: Check if pnpm generates a new package-lock.json

File: src/pages/TaskDetails/TaskDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { CardContent, Typography } from '@mui/material'
+import Card from '@mui/material/Card'
+
+// de la url un id, y ese de id muestra sus detalles;
```
Comment (Alan-TheGentleman) [inline src/pages/TaskDetails/TaskDetails.tsx]: Comments should be deleted before asking for a pr

File: src/pages/TaskDetails/TaskDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+// una tarea 3 atributos:  id, name, description
+
+const task = {
+  id: 686,
+  name: 'First Task',
+  description: 'This is the first task',
+}
+
+export const TaskDetails: React.FC = () => {
```
Comment (Alan-TheGentleman) [inline src/pages/TaskDetails/TaskDetails.tsx]: Don't need React.FC as is just a function that we will use later as a component, it will be infered

**Review by Alan-TheGentleman — CHANGES_REQUESTED** [review]

**Review by Alan-TheGentleman — APPROVED** [review]
LGTM !

**Comment by NeilChavez** [comment]
Thank you Alan :)

**Pull request merged**